webforces/server/interface/dbworker.py

```diff
@@ -9,3 +10,44 @@ def connect(self) -> int:
     @abc.abstractmethod
     def disconnect(self) -> int:
         pass
+
+    @abc.abstractmethod
+    def dropAll(self) -> None:
+        pass
+
+    @abc.abstractmethod
+    def addUser(self, user) -> (int, User):
```

Comment:
-> (DBStatuses, User)
Same for other functions.

Reply:
Thank you! Changed.